assets/js/blocks/catalog-sorting/style.scss

```diff
@@ -3,8 +3,8 @@
 		float: none;
 	}
 
-	select {
+	select.orderby {
```

Comment:
For some reason, the `<select>` in the editor doesn't have the `orderby` class for me. So the styles are not applied:

![imatge](https://user-images.githubusercontent.com/3616980/216553959-e448b4d9-22f8-42d1-b1ce-3aeab552842c.png)

In the frontend, everything works well. Can you reproduce?

Reply:
@Aljullu Ah, I forgot testing in the editor. I fixed this issue.